Marlin/src/pins/stm32f4/pins_BTT_GTR_V1_0.h

```diff
@@ -101,11 +101,11 @@
 //
 // Pins on the extender
 //
-//#define X_MIN_PIN                         PI4
-//#define X2_MIN_PIN                        PF12
-//#define Y_MIN_PIN                         PF4
-//#define Y2_MIN_PIN                        PI7
-//#define Z_MIN_PIN                         PF6
+//#define X_MIN_PIN                         PI4   // M5 M1_STOP
+//#define X2_MIN_PIN                        PF12  // M5 M5_STOP
+//#define Y_MIN_PIN                         PF4   // M5 M2_STOP
+//#define Y2_MIN_PIN                        PI7   // M5 M4_STOP
+//#define Z_MIN_PIN                         PF6   // M5 M3_STOP
```

Comment:
Do you know why these are defined like this at all?

These will conflict with the definitions above in at least some situations...
I suspect these are here because _one person_ used them this way?

Reply:
Anything after // doesn't affect the code. They are just comments for those definitions, so anyone adding an M5 would know what's what. Like Marlin code using E3 witch is M5-Motor1. They use E3-7 to be more compatible with other boards instead of making custom code for each board manufacturer. 

If you look at the changes, red is the old lines, and green are the new changes.

Reply:
I realize all of this, and that they are commented out. I just wonder why these would have ever been defined for MIN endstops for drivers which already exist on the mainboard.

Reply:
@NAPCAL — It would be better to add a custom define for the extender, then set pins based on that, rather than simply having them commented out here.

Reply:
@thinkyhead Do you mean to add a define for if the M5 will be used to enable all of the M5 pin assignments?

**Example 1**
Place this around line 33
//#define M5_BOARD // uncomment if using M5 connected to the GTR


Place this around line 307
#ifdef M5_BOARD
 #undef BOARD_INFO_NAME
 #define BOARD_INFO_NAME "BTT GTR/M5 V1.0"

   * place here all of the defines for the M5 connections

#endif

**Example 2** moving all M5 pin assignments to a pins file.
Place this starting around line 33

//#define M5_BOARD // uncomment if using M5 connected to the GTR

#ifdef M5_BOARD
 #include "pins_BTT_M5.h"
#endif